requirements.txt

```diff
@@ -0,0 +1,17 @@
+# For DateTime manipulation
+# Version is latest at time of writing
+DateTime==4.1.1
+pytz==2016.6.1
+
+# For HTTP requests to Catchpoint Pull API
+# Version is latest at time of writing
+requests==2.11.1
+
+
+# Transitive libraries
+# This is needed so there are no version conflicts when
+# one downstream library does NOT specify the version it wants,
+# and another one does.
+
+# From DateTime
+zope.interface==4.3.2
```

Comment:
It is out convention, but I am forking someone else's repo in hopes of getting an upstream merge. So, trying to follow the mainstream conventions rather than Krux's.